internal/k3d/terraform.go

```diff
@@ -53,3 +59,48 @@ type GithubTerraformEnvs struct {
 	AwsAccessKeyId        string
 	AwsSecretAccessKey    string
 }
+
+
+func TerraformPrep(config *K3dConfig) error{
+
+	path := config.GitopsDir + "/terraform"
+	log.Info().Msgf("Repooo is %s",path)
+	err := filepath.Walk(path,detokenizeterraform(path,config))
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func detokenizeterraform(path string,config *K3dConfig) filepath.WalkFunc {
+	return filepath.WalkFunc(func(path string,fi os.FileInfo,err error) error{
+
+		if fi.IsDir()  {
+			return nil
+		}
+
+		matched,_ := filepath.Match("*",fi.Name())
+
+		if matched {
+
+			read,err := ioutil.ReadFile(path)
+			if err != nil {
+				return err
+			}
+
+			newContents := string(read)
+			newContents = strings.Replace(newContents,"<ADMIN_TEAM>",config.AdminTeamName,-1)
+			newContents = strings.Replace(newContents,"<DEVELOPER_TEAM>",config.DeveloperTeamName,-1)
+			newContents = strings.Replace(newContents,"<METAPHOR_REPO_NAME>",config.MetaphorRepoName,-1)
+			newContents = strings.Replace(newContents,"<GIT_REPO_NAME>",config.GitopsRepoName,-1)
+
+			err = ioutil.WriteFile(path,[]byte(newContents),0)
+			if err != nil {
+				return err
+			}
```

Comment:
This would also benefit of my recommendations above:

* Use `strings.NewReplacer()` for better readability
* Error wrapping with `fmt.Errorf()` and the `%w` directive
* Removing the deprecated package call to `ioutil` and replace it with whatever's not deprecated